docs/docs/topics/mutual-auth.md

```diff
@@ -0,0 +1,306 @@
+
+This is a long page that dives into several specific security practices that provide mutual authentication. You can use the table of contents below to narrow down to the specific tools you're interested in, or read the entire doc for a deeper understanding of how these tools work together to support strong infrastructure security.
+
+[[toc]]
+
+## What Is Mutual Authentication?
+
+The concept of mutual authentication is deceptively simple. It means that both sides of a connection can validate the identity of the other. The application of this concept however, can be varied and complex. Pomerium provides the features and capability to provide mutual authentication between itself and end users, as well, as between itself and upstream services, but configuring those external endpoints can vary depending on each service's features.
+
+## Pomerium Defaults
+
+> - **Security**: Moderate
+> - **Difficulty**: Easy
+> - **Encrypted**: Yes (from the end user to Pomerium. It's up to the service to provide a TLS endpoint for Pomerium to use.)
+> - **Mutual Authentication**: None
+
+Let's look at a basic installation of Pomerium on a local network, with a single downstream service. This service contains sensitive data that we only want the right people to access.
+
+::: tip Note
+This section describes multiple services of Pomerium like Authenticate and Proxy as separate services. If you are running Pomerium in all-in-one mode, these will be a single process/container in your environment.
+:::
+
+```mermaid
+flowchart TD
+    D[End User]
+        D <--> A
+    subgraph LAN
+    style LAN stroke-dasharray: 5 5
+    A(Pomerium Proxy Service)
+    B(Pomerium Authenticate Service)
+    A <-.-> B
+    C(Service)
+    A <--> C
+    end
+```
+
+1. When the user connects to the route `service.example.com` the traffic is received by the Pomerium Proxy service.
+1. The proxy redirects to the Authenticate service to validate the user by having them sign in with the identity provider (**IdP**).
+1. After authentication succeeds, the proxy verifies that the authenticated (**authn**) user is authorized (**authz**) to access the service, and begins communication with it.
+
+This is a good start, only the users that are supposed to have access to the service get through. But this model is dependent on the security of your network perimeter. If a bad actor gains access to your local area network (**LAN**), they can now communicate with the service directly:
+
+```mermaid
+flowchart TD
+    D[End User]
+    D <--> A
+    E[/Hacker/]
+    E<--->C
+    subgraph LAN
+    style LAN stroke-dasharray: 5 5
+    A(Pomerium Proxy Service)
+    B(Pomerium Authenticate Service)
+    A <-.-> B
+    C(Service)
+    A <--> C
+    end
+```
+
+While your network *should* be secured to only allow traffic at specified ports and directed to specified services, this creates a single point of failure. A hacker need only bypass your firewall to get direct access to your service.
```

Comment:
>> A hacker **need** only bypass your firewall to get direct access to your service.

I think the verb is wrong here.

Reply:
> However, in a rather formal style of English, there also exists an auxiliary verb ("helping verb") need, which is followed by a bare infinitive, and is only used in negative polarity contexts: with not, with only, in questions, and so on. Additionally, it only exists in the present tense. Aside from these two restrictions, it's very much like other auxiliary verbs (can, may, should, etc.): "We need not discuss it."
> 
> So your first example ("[…] one need only look […]") is correct — using the auxiliary verb need — whereas your second example ("[…] one needs only look […]") is wrong — it should either be "one need only look" (using the auxiliary verb), or "one only needs to look" (using the regular verb).

From [english.stackexchange](https://english.stackexchange.com/questions/297233/is-one-needs-only-or-one-need-only-correct).

Reply:
TIL thanks for the explanation.